README.md

```diff
@@ -41,6 +41,15 @@ Some applications load from `.env`, `.env.local`, `.env.development` and `.env.d
 (see [#37](https://github.com/entropitor/dotenv-cli/issues/37) for more information).
 `dotenv-cli` supports this using the `-c` flag for just `.env` and `.env.local` and `-c development` for the ones above.
 
+### Override
+Override any environment variables that have already been set on your machine with values from your .env file.
+
+Example:
+```bash
+$ echo "NODE_ENV=test" > .env
+$ NODE_ENV=production dotenv --override -p LOG_LEVEL
+test
```

Comment:
Maybe good to also give an example of what it would give without the override flag? 


```suggestion
# By default, dotenv-cli will not override env variables already set
$ NODE_ENV=production dotenv -p NODE_ENV
production
# You can opt-in to this behavior using `--override`
$ NODE_ENV=production dotenv --override -p NODE_ENV
test
```

Notice that you had LOG_LEVEL <> NODE_ENV in your example